src/environment.js

```diff
@@ -69,7 +69,9 @@ export const sortAppsPair = (app1, app2) => {
 }
 
 // Use appOverrides to override specific keys in an app instance, e.g. the start_url or script location
-const appOverrides = {}
+const appOverrides = {
+  [appIds['TokenManager']]: { name: 'Tokens' },
+}
```

Comment:
A bit hesitant on this, but I can see the rationale.

Just wanted to check @dizzypaty; this will make it so all 0.6 organizations also have their Token Managers get renamed to Token in the sidebar, but the app's UI itself won't be changed.

Reply:
@delfipolito @sohkai I think this is ok for now. I imagine that with the 0.8 release, all apps UI's will be also updated and the Token Manager title will change to Tokens, right?

Reply:
We will never have the ability to change old versions of apps (particularly, only 0.6 orgs would have this issue), but hopefully by then most users will have upgraded.